### PR TITLE
Update python code to dynamic-graph-python switch to boost-python.

### PR DESCRIPTION
**Pull request by florent-lamiraux**

### CODE REVIEW

File: scripts/simulation.py [hunk 1]
```diff
@@ -147,7 +147,8 @@ def initializeSoT2HPPconversion (self):
         from agimus_sot_msgs.srv import GetJointNames
         get_joint_names = ros_tools.wait_for_service("/agimus/sot/get_joint_names", GetJointNames)
         self.sot_joint_names = get_joint_names().names
-        assert self.sot_joint_names[0] == "root_joint"
+        # Assert that first joint name ends by "root_joint"
+        assert self.sot_joint_names[0][-10:] == "root_joint"
```
Comment (jmirabel) [inline scripts/simulation.py]: Maybe
```suggestion
        assert self.sot_joint_names[0].endswith("root_joint")
```

File: src/agimus_sot/control/gripper.py [hunk 2]
```diff
@@ -61,22 +61,20 @@ def _makeTorqueControl (self, nums, denoms):
     def _makeIntegrationOfVelocity (self):
         from dynamic_graph.sot.core.operator import Add_of_vector
         self.omega2theta = Add_of_vector (self.name + "_omega2theta")
-        self.omega2theta.setCoeff2(self.dt)
-        # self.omega2theta.sin1 <- current position
-        # self.omega2theta.sin2 <- desired velocity
-        plug (self.torque_controller.output, self.omega2theta.sin2)
+        self.omega2theta.coeffs = np.array([0, self.dt])
```
Comment (jmirabel) [inline src/agimus_sot/control/gripper.py]: Shouldn't it be
```suggestion
        self.omega2theta.coeffs = np.array([1, self.dt])
```

Comment (florent-lamiraux) [reply src/agimus_sot/control/gripper.py]: I do not know. I did not find any initialization for the first coefficient.

Comment (jmirabel) [reply src/agimus_sot/control/gripper.py]: I think they were initialized with ones. The operation that this does is
image

**Pull request merged**